src/main/scala/com/codacy/plugins/utils/BinaryDockerHelper.scala

```diff
@@ -0,0 +1,84 @@
+package com.codacy.plugins.utils
+
+import com.codacy.plugins.api.PatternDescription
+import com.codacy.plugins.runners.{DockerConstants, IDocker}
+
+import java.nio.file.{Files, Path}
+import scala.util.{Properties, Try}
+
+class BinaryDockerHelper extends DockerHelper {
```

Comment:
Similar to the other added files, are we adding because it got removed or something? Was this code on codacy-plugins before but now is not available?